sign_language_datasets/datasets/chicagofswild/chicagofswild.py

```diff
@@ -39,14 +43,20 @@
 _CHICAGO_FS_WILD_PLUS_URL = "https://dl.ttic.edu/ChicagoFSWildPlus.tgz"
 _CHICAGO_FS_WILD_URL = "https://dl.ttic.edu/ChicagoFSWild.tgz"
 
+_POSE_URLS = {"holistic": "/shares/volk.cl.uzh/zifjia/ChicagoFSWild/pose/"}
```

Comment:
@AmitMY shall we determine a location on S3IT to host the poses locally, or are you keen on making them on the cloud? I do not have access to the latter.

Reply:
If you compress the directory into one file (gz, tar, whatever you want), I can upload it to google cloud

Reply:
Sounds good.